frontend/package.json

```diff
@@ -27,6 +27,8 @@
     "test-gui": "yarn run test-suite --suite all",
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "test-cypress": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
+    "test-cypress-devconsole": "cd packages/dev-console/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-devconsole-headless": "cd packages/dev-console/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser chrome --headless --spec \"features/project-creation.feature\";",
     "test-cypress-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
```

Comment:
Hi, I think for future maintenance, as more teams start creating their cypress tests, we should set a good precidence.  Can you pls copy "test-cypress-headless" line to "test-cypress-console-headless", then create a new "test-cypress-headless": `yarn run test-cypress-console-headless && test-cypress-devconsole-headless`. (note: you might have to play around with `cd ../../` between the `&&`'s as the sub-runs expect to run from `frontend/package`)

Then when `olm` creates their cypress tests, it would be a natural follow on:  "test-cypress-headless": `yarn run test-cypress-console-headless && test-cypress-devconsole-headless && test-cypress-olm-headless`.

This would also bypass having to update `test-cypress.sh` which calls `yarn run test-cypress-headless` and is called by our `test-prow-e2e.sh` during PR CI's.
cc @harishsurf

Reply:
updated